src/Logary.Facade/Facade.fs

```diff
@@ -297,6 +297,14 @@ type LoggingConfig =
     /// adapter to replace this semaphore with a global semaphore.
     consoleSemaphore: obj }
 
+[<AutoOpen>]
+module internal Helpers = 
```

Comment:
I don't think this should go here; it's not for Logary to provide backwards-compatibilities; if the author using the facade wants this he'll import a shim of his own and place it ahead of the facade. Plus, you don't capture the full extent of isNull **I think**, or else the recommended way would be `if o = null` by the same logic you have in your implementation.

Reply:
No probs, I’m only moving it here, I didn’t add it. I can remove it a bit later when I’m back at the computer, but it came from here originally https://github.com/logary/logary/commit/fb7f4398f4895a7c02483d08163ac1aeee737a66